spec/integration/shared_examples/volume/api500/create.rb

```diff
@@ -231,18 +231,30 @@
   end
 
   describe '#set_storage_pool' do
+    it 'set_storage_pool in properties' do
+      item = described_class.new(current_client, name: VOLUME_NAME, properties: {})
+      item.set_storage_pool(storage_pool)
+      expect(item['properties']['storagePool']).to eq(storage_pool['uri'])
+    end
+
     it 'set_storage_pool' do
       item = described_class.new(current_client, name: VOLUME_NAME)
       item.set_storage_pool(storage_pool)
-      expect(item['properties']['storagePool']).to eq(storage_pool['uri'])
+      expect(item['storagePoolUri']).to eq(storage_pool['uri'])
```

Comment:
But the result of `set_storage_pool ` should have set the `['properties']['storagePool']`, because it is the test of create action, right?! For create action, the URI of storage pool should be inside `item['properties']['storagePool']`.

Reply:
no problem. That things happen. 😄